exporters/stdout/stdoutlog/config.go

```diff
@@ -0,0 +1,85 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package stdoutlog // import "go.opentelemetry.io/otel/exporters/stdout/stdoutlog"
+
+import (
+	"io"
+	"os"
+)
+
+var (
+	defaultWriter      = os.Stdout
+	defaultPrettyPrint = false
+	defaultTimestamps  = true
+)
+
+// config contains options for the STDOUT exporter.
+type config struct {
+	// Writer is the destination.  If not set, os.Stdout is used.
+	Writer io.Writer
+
+	// PrettyPrint will encode the output into readable JSON. Default is
+	// false.
+	PrettyPrint bool
+
+	// Timestamps specifies if timestamps should be printed. Default is
+	// true.
+	Timestamps bool
+}
+
+// newConfig creates a validated Config configured with options.
+func newConfig(options ...Option) (config, error) {
```

Comment:
This will be called from a function that has a slice of `Option`. It should accept the slice directly to avoid the additional `...`.

```suggestion
func newConfig(options []Option) (config, error) {
```